index.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap" rel="stylesheet">
+  <title>capston</title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <section class="first">
+  <header>
+    <div class="flists">
+      <i class="fa-brands fa-facebook-f"></i>
+      <i class="fa-brands fa-facebook-f"></i>
+      <ul class="flist">
+        <li><a href="">English</a></li>
+        <li><a href="">My page</a></li>
+        <li><a href="">logout</a></li>
+      </ul>
+    </div>
+    <div class="bar" id="bar">
+      <i class="fa-solid fa-bars fa-2x"></i>
+    </div>
+    <div class="hide" id="close">
+      <i class="fa-solid fa-xmark "></i>
+    </div>
+    <nav class="lnav">
+      <div class="logo">
+        <h1><a href="#hello">AYC</a></h1>
```

Comment:
- [ ] Please remove the `<h1>` tag that wraps the logo link for semantic reasons because this is not your page's main header.